spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -917,6 +1026,7 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde {
                 .setStringSpace(builder)
                 .build())
           } else {
+            withInfo(expr, null, child)
```

Comment:
Can we override this method for this kind of usage so we can do `withInfo(expr, child)`?

Can be in a follow up PR.